spec/system/help_page_spec.rb

```diff
@@ -25,4 +25,26 @@
       expect(page).not_to have_link "Help"
     end
   end
+
+  scenario "renders SDG help page link when feature is enabled" do
+    Setting["feature.help_page"] = true
+
+    Setting["feature.sdg"] = true
+    visit root_path
```

Comment:
Maybe we can group the setup together and leave a blank line after it:

```suggestion
    Setting["feature.help_page"] = true
    Setting["feature.sdg"] = true
    
    visit root_path
```